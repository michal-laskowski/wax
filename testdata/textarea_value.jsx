export function textarea_value() {
    const value = "<div>test area expected value</div>";
    return (
        <>
            <textarea value="abc" />
            <textarea >{`a&b"c`}</textarea>
            <textarea value={`a&b"c`} />
            <textarea value="true" />
            <textarea value={value} />
            <textarea value={true} />
            <textarea value="" />
            <textarea value={""} />
            <textarea value={"    "} />
            <textarea value={"a\nb"} />
            <textarea value={"c\nd"}></textarea>
            <textarea>{"e\nf"}</textarea>
            <textarea>{"a<></>b`"}</textarea>
        </>
    );
}