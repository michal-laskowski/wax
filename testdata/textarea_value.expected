<textarea>abc</textarea>
<textarea>a&amp;b&#34;c</textarea>
<textarea>a&amp;b&#34;c</textarea>
<textarea>true</textarea>
<textarea>&lt;div&gt;test area expected value&lt;/div&gt;</textarea>
<textarea></textarea>
<textarea></textarea>
<textarea></textarea>
<textarea>    </textarea>
<textarea>a
b</textarea>
<textarea>c
d</textarea>
<textarea>e
f</textarea>
<textarea>a&lt;&gt;&lt;/&gt;b`</textarea>